background.js

```diff
@@ -0,0 +1,24 @@
+function onBeforeRequest(data){
+    //perhaps put 'data' in parameter
+    console.log("top of on before request method");
+    console.log(data);
+    //need to return something
+    //this is where blocking goes and also figuring out whether something needs to be blocked
+    var urlNotFixed = window.location.toString();
+    urlNotFixed.replace(" ", "");
+    urlNotFixed.toLowerCase();
+    var urlFixed = urlNotFixed;
+    console.log(urlFixed);
+    var blockedList = "facebook.com";
+    if(urlFixed.contains(blockedList)){
+        return webRequest.BlockingResponse;
```

Comment:
`webRequest.BlockingResponse` is the type that Chrome expects as the return value, so what we need to do is return an instance of that type, not the type itself. That is, we need to return an object that has the properties that an object of `BlockingResponse` type can or should have. Take a look at https://developer.chrome.com/extensions/webRequest#type-BlockingResponse; there you can see what properies a `BlockingResponse` object can have. You can pass a value just for the one you need to achieve your goal, since all are optional ~